ansible/roles/ocp-workload-edge-deployments/tasks/remove_workload.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Implement your Workload removal tasks here
+- name: Cleaning up the environment
+  debug:
+    msg: "Cleaning up the environment.."
+
+- name: Delete project IoT Development
+  shell: "oc delete project iot-development"
+  ignore_errors: true
+
+- name: Delete project IoT Testing
+  shell: "oc delete project iot-testing"
+  ignore_errors: true
+
+- name: Delete project IoT Hub
+  shell: "oc delete project iot-hub"
+  ignore_errors: true
+
+- name: Delete BuildConfig in openshift namespace
+  shell: "oc delete bc deploy-containers-apb -n openshift"
+  ignore_errors: true
+
+- name: Delete ImageStream in openshift namespace
+  shell: "oc delete is deploy-containers-apb -n openshift"
+  ignore_errors: true
+
+- name: Cleaning up temp dir
+  shell: "rm -rf /tmp/{{guid}}"
+  ignore_errors: true
+
+- name: Restoring up /etc/containers/registries.conf
+  become: true
+  shell: mv /etc/containers/registries.conf.bak /etc/containers/registries.conf
+  ignore_errors: true
```

Comment:
i'm not a big fan of all those `ignore_errors`. It looks like cheap idempotency.
They will appear as error in the logs, and will catch the eyes if someone is trying to troubleshoot.

Ideally (if time permits) i would test for existence before trying to delete.

Reply:
What do you suggest? Continue using `shell` module in a way like this?
`oc get projects|grep iot-development && oc delete project iot-development`

Or using [register variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#register-variables)?

Reply:
"register variables" is a more common ansible pattern.